src/packages/@ncigdc/modern_components/SsmsTable/SsmsTable.model.js

```diff
@@ -300,8 +332,8 @@ const SsmsTableModel = [
                     'consequence.hits.edges[0].node.transcript.gene.symbol'
                   )} ${get(
```

Comment:
this is the actual change, everything else is formatting

Reply:
It feels `get` should let us pass it alternate variables to be found, rather than having to 'get' multiple times... right?

Reply:
yeah, it takes a third arg for a default, but doesn't allow that to be another `get`...unless there is another way of using this i'm missing